src/auslib/web/public/client.py

```diff
@@ -230,6 +230,8 @@ def get_update_blob(transaction, **url):
     response = make_response(xml)
     response.headers["Cache-Control"] = app.cacheControl
     response.headers.extend(get_aus_metadata_headers(eval_metadata))
+    if query["product"] in app.config.get("CONTENT_SIGNATURE_PRODUCTS", []):
+        response.headers.extend(get_content_signature_headers(xml, True))
```

Comment:
There's a bit of a disconnect here, where I say, "if this is any of the products that require signing, sign with the GMP autograph". Having a per-product table of autograph servers seems unwarranted at this time, but I'm open to making this more flexible if you think otherwise.

Reply:
The other way to do this would be to pass a product name instead of a `is_gmp` bool -- and always look up by that product name. This might be a bit more future proof without needing to maintain a table of all of the products. I'm okay either way though - up to you.

Reply:
Yes, let's do that.